crates/extensions/c8y_mapper_ext/src/operations/convert.rs

```diff
@@ -433,7 +434,17 @@ impl CumulocityConverter {
             serde_json::Value::Object(serde_json::Map::new())
         };
 
-        let request = GenericCommandState::new(topic, CommandStatus::Init.to_string(), payload);
+        let mapper_id = self.command_id.prefix();
+        let inject_object = json!({
+            "status": "init",
```

Comment:
As noticed by @rina23q, this `status` has to be injected because of an unexpected behavior of `GenericCommandState::update_with_json` which expect a status in the injected value and inject an error status if none is provided.

=> I will push a fix to avoid this surprising behavior.

Reply:
Here is the fixup commit: https://github.com/thin-edge/thin-edge.io/pull/3283/commits/e9eb073e9546cc64c64cd2be590a2d47844eb9d5.

In fact the issue was in `GenericCommandState::new()` which was not properly injecting the state in the paylaod.